src/track/cueinfo.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+// cueinfo.h
+// Created 2020-02-28 by Jan Holthuis
+
+#include "util/color/rgbcolor.h"
+
+namespace mixxx {
+
+enum class CueType {
+    Invalid = 0,
+    HotCue = 1,
+    MainCue = 2,
+    Beat = 3, // unused (what is this for?)
+    Loop = 4,
+    Jump = 5,
+    Intro = 6,
+    Outro = 7,
+    AudibleSound = 8, // range that covers beginning and end of audible
+                      // sound; not shown to user
+};
+
+static constexpr double kNoPosition = -1.0;
```

Comment:
Please add a comment that `kNoPosition` is used for both positions in milliseconds as well as positions in samples!

I would like to ban `kNoPosition` from this class and instead use std::optional<double> to be explicit. But I don't know how much effort this would be?

Reply:
The std::optional remark also applies to `kNoHotCueNumber`. I leave it up to you to make a decision.

Reply:
I can do that, but only for this class, not the whole codebase ;-)